.meta/sinks/humio.toml

```diff
@@ -0,0 +1,49 @@
+[sinks.humio]
+title = "Humio"
+batch_max_size = 1049000
+batch_timeout_secs = 1
+buffer = true
+common = false
+beta = true
+delivery_guarantee = "at_least_once"
+egress_method = "batching"
+function_category = "transmit"
+healthcheck = true
+input_types = ["log"]
+request_rate_limit_duration_secs = 1
+request_rate_limit_num =10
+request_retry_attempts = 5
+request_retry_initial_backoff_secs = 1
+request_retry_max_duration_secs = 10
+request_in_flight_limit = 10
+request_timeout_secs = 60
+write_to_description = "[Humio][urls.humio] via the [HEC API][urls.humio_hec]"
```

Comment:
Maybe a description that says "via the [Vector API][urls.humio_vector], currently based on the [HEC API]."

Reply:
so here is what the preview currently looks like for the docs https://5e3b261c9b5eaa000a7732c0--vector-project.netlify.com/docs/reference/sinks/humio/ 

The idea is that this `write_to_description` would be what comes after the `to` part in that doc string. So I am not sure it makes sense to mention vector again there? what do you think?